src/main/scala/higherkindness/rules_scala/workers/zinc/compile/AnalysisStore.scala

```diff
@@ -105,20 +110,122 @@ class AnxAnalyses(format: AnxAnalysisStore.Format) {
   private[this] val reader = new ProtobufReaders(mappers.getReadMapper, schema.Version.V1)
   private[this] val writer = new ProtobufWriters(mappers.getWriteMapper)
 
+  def sortAnnotation(annotation: Annotation): Annotation = {
+    annotation.copy(
+      arguments = annotation.arguments.sortWith(_.hashCode() > _.hashCode())
+    )
+  }
+
+  def sortTypeParameter(typeParameter: TypeParameter): TypeParameter = {
+    typeParameter.copy(
+      annotations = typeParameter.annotations.map(annotation => sortAnnotation(annotation)).sortWith(_.hashCode() > _.hashCode()),
```

Comment:
Is the second `sortWith` here sorting twice?

Reply:
There's already a sort inside `sortAnnotations`

Reply:
I think that this `sortWith` is sorting the list of annotations, whereas `sortAnnotations` is sorting the objects within a specific annotation